netlogo-gui/src/main/app/WidgetPanel.scala

```diff
@@ -625,14 +620,6 @@ class WidgetPanel(val workspace: GUIWorkspace)
       component.setSize(size)
   }
 
-  /// buttons
-
-  protected def loseFocusIfAppropriate(): Unit = {
```

Comment:
I was. That was a silly question - given that it's already been deleted! Nothing more to be done here!